tools/build/third_party/breakpad/windows.patch

```diff
@@ -0,0 +1,80 @@
+diff --git a/src/common/dwarf/dwarf2reader.cc b/src/common/dwarf/dwarf2reader.cc
+index f2dc4afe..56bbe98a 100644
+--- a/src/common/dwarf/dwarf2reader.cc
++++ b/src/common/dwarf/dwarf2reader.cc
+@@ -64,7 +64,7 @@ CompilationUnit::CompilationUnit(const string& path,
+       is_split_dwarf_(false), dwo_id_(0), dwo_name_(),
+       skeleton_dwo_id_(0), ranges_base_(0), addr_base_(0),
+       have_checked_for_dwp_(false), dwp_path_(),
+-      dwp_byte_reader_(), dwp_reader_() {}
```

Comment:
Why?

Reply:
The dwp_reader is only used for "split dwarf" and assumes ELF. We don't need split dwarf.